main.go

```diff
@@ -1,8 +1,31 @@
 package main
 
-import "github.com/Brightscout/mattermost-load-test-scripts/scripts"
+import (
+	"fmt"
+	"os"
+
+	"github.com/Brightscout/mattermost-load-test-scripts/scripts"
+)
 
-// TODO: remove later, if not needed
 func main() {
-	scripts.LoadConfig()
+	config, err := scripts.LoadConfig()
+	if err != nil {
+		panic(err)
+	}
+
+	args := os.Args
+	if len(args) > 1 {
+		switch args[1] {
+		case "create_users":
```

Comment:
constants for these

Reply:
I don't think these will be used anywhere else. We can create a constant later if needed.

Reply:
but it would be nice to have all the functionalaties like create_users etc at one place